doc/release/release_notes.rst

```diff
@@ -4,6 +4,28 @@
 Release Notes
 *************
 
+.. release:: Upcoming
+
+    .. change:: changed
+
+        Added compatibility with python 3.6, 3.7 and 3.8.
+
+    .. change:: fixed
+
+        Fixed :class:`wiz.graph.Resolver` to ensure that conflicted nodes are
+        always sorted in ascending order of distance from the :attr:`root
+        <Graph.ROOT>` level of the graph.
+
+        Previously, conflicting nodes wouldn't be sorted properly when new
+        packages are added to the graph during the conflict resolution process.
+
+    .. change:: fixed
+
+        Fixed :func:`wiz.utility.extract_version_ranges`.
```

Comment:
fixed what in it? The comment in the code says "Sort specifiers for deterministic results.", so

```suggestion
        Fixed :func:`wiz.utility.extract_version_ranges` to sort specifiers
        properly for deterministic results.
```
?